DigitalLearningSolutions.Web/ViewModels/Common/MiniHub/MiniHubSection.cs

```diff
@@ -0,0 +1,9 @@
+namespace DigitalLearningSolutions.Web.ViewModels.Common.MiniHub
+{
+    public class MiniHubSection
+    {
+        public string ControllerName { get; set; }

```

Comment:
Should we have these as get-only properties and set them in a constructor (or are we expecting to change them after setting them in the first place)?